<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>EditorForm</class>
 <widget class="QWidget" name="EditorForm">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1022</width>
    <height>688</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QTextEdit" name="textEdit">
   <property name="geometry">
    <rect>
     <x>30</x>
     <y>170</y>
     <width>291</width>
     <height>241</height>
    </rect>
   </property>
  </widget>
  <widget class="QFrame" name="toolFrame">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>50</y>
     <width>631</width>
     <height>80</height>
    </rect>
   </property>
   <property name="frameShape">
    <enum>QFrame::Shape::StyledPanel</enum>
   </property>
   <property name="frameShadow">
    <enum>QFrame::Shadow::Raised</enum>
   </property>
   <widget class="QFontComboBox" name="fontCombo">
    <property name="geometry">
     <rect>
      <x>10</x>
      <y>30</y>
      <width>255</width>
      <height>28</height>
     </rect>
    </property>
    <property name="editable">
     <bool>false</bool>
    </property>
    <property name="currentIndex">
     <number>172</number>
    </property>
   </widget>
   <widget class="QDoubleSpinBox" name="textSpin">
    <property name="geometry">
     <rect>
      <x>320</x>
      <y>30</y>
      <width>44</width>
      <height>29</height>
     </rect>
    </property>
   </widget>
   <widget class="QFrame" name="formatFrame">
    <property name="geometry">
     <rect>
      <x>400</x>
      <y>20</y>
      <width>177</width>
      <height>51</height>
     </rect>
    </property>
    <property name="frameShape">
     <enum>QFrame::Shape::StyledPanel</enum>
    </property>
    <property name="frameShadow">
     <enum>QFrame::Shadow::Raised</enum>
    </property>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <widget class="QPushButton" name="boldButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::FormatTextBold"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="italicsButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::FormatTextItalic"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="underlineButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::FormatTextUnderline"/>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QPushButton" name="strikeButton">
       <property name="text">
        <string/>
       </property>
       <property name="icon">
        <iconset theme="QIcon::ThemeIcon::FormatTextStrikethrough"/>
       </property>
      </widget>
     </item>
    </layout>
   </widget>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
